docs/usage/index.rst

```diff
@@ -1,9 +1,10 @@
 Usage
 =====
 
-.. include:: with-our-python-api.rst
-.. include:: from-a-terminal.rst
+.. include:: bash-terminal.rst
```

Comment:
Please keep order.

- Python API First
- Terminal (please reverse filename as it is in many bookmarks already).
- Deprecated Variables
- GitHub Action
- GitLab CI/CD
- Travis CI

Reply:
As it makes most seance with the new name I'll rather suggest a "empty" `from-a-terminal.rst` which either

  - informs of new location
  - Include the 'bash-terminal.rst'

Reply:
A powershell is a terminal.
A bash is a terminal. 
PyFunceble can be used in both.

Reply:
> A powershell is a terminal.
> A bash is a terminal.
> PyFunceble can be used in both.

True, but also the very same syntax you are using in bash scripting...  Which is the intention for renaming the file, to have it's name indicating that this is not ONLY for CLI command but also highly programmable commands too.

I'll give it another thought when it no longer is `To Hot To Be Human`

Reply:
A brainstorm for handling the naming of `from-a-terminal.rst` to something more corvering of terminal + scripting

 - `terminal-scripting.rst`
 - `scripting-terminal.rst`
 - ??

Another alternative could be to split it into 3 files

  - scripting-sh.rst
    - .. include::args.rst
  - scripting-ps.rst
    - .. include::args.rst
  - terminal.rst
    - .. include::args.rst
  - args.rst
  
This will in the long run make it possible to make some more specific and in-depth explanation related to each of the different processes as IE Pyfunceble in bash/sh scripts is running as a sub-process and hence arguments should be parsed as uppercase

.. NOTE:: I'm really going for a rename of `from-a-terminal.rst` as it have become to "OFF" in its name compared to the rest of the dirs files.

New title: (Line 12)

```rst
Usage from a terminal or for scripting
-------------------------------------------------
```

Reply:
Mostly to self + @funilrys 

We should consider this one before switching from beta

https://mypdns.org/spirillen/PyFunceble/-/issues/26

Reply:
Mhhhh `terminal.rst` is okay for me.

Reply:
> Mhhhh `terminal.rst` is okay for me.

- https://mypdns.org/spirillen/PyFunceble/-/commit/36dca1f56608d87c79056c5e8207ec792fa6e9cd
- https://mypdns.org/spirillen/PyFunceble/-/commit/67869331cd420a6924fb5830e20ad206899aee3b
- https://mypdns.org/spirillen/PyFunceble/-/commit/e5c687bbdc9968dd2c37ac2750674a297a088dff